internal/util/assertmetrics/assert_metrics.go

```diff
@@ -0,0 +1,45 @@
+package assertmetrics
```

Comment:
Can this file move to a `_test.go` file?

Reply:
I don't think you can depend on `_test.go` files in tests of other packages... Similar to `testappender` package.